package.json

```diff
@@ -54,6 +54,7 @@
     "heap": "~0.2.6",
     "interface-datastore": "^5.1.1",
     "it-first": "^1.0.4",
+    "it-length": "^1.0.3",
     "it-length-prefixed": "^5.0.2",
     "it-pipe": "^1.1.0",
     "k-bucket": "^5.1.0",
```

Comment:
we want to get rid of `xor-distance` below, right? 🎉

Reply:
Done!